scripts/ci_steps/prepare_protocol_and_unit_tests.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 # Regenerate the SDK Package.swift to run only protocol tests and runtime unit tests
```

Comment:
I don't think we need this comment at all anymore

Reply:
Deleted as suggested.